src/libs/actions/Policy.js

```diff
@@ -214,12 +214,12 @@ function hasActiveFreePolicy(policies) {
 }
 
 /**
- * Build optimistic data for adding users from announce room
+ * Build optimistic data to the announce room
```

Comment:
I was expecting us to keep the "for adding users" part. `Build optimistic data for adding members to the announce room`